br/pkg/lightning/restore/chunk_restore_test.go

```diff
@@ -100,13 +109,13 @@ func (s *chunkRestoreSuite) TestDeliverLoopEmptyData(c *C) {
 		Return(nil).AnyTimes()
 
 	dataEngine, err := importer.OpenEngine(ctx, &backend.EngineConfig{}, s.tr.tableName, 0)
-	c.Assert(err, IsNil)
+	require.Nil(s.T(), err)
```

Comment:
```suggestion
	require.NoError(s.T(), err)
```

Reply:
ditto others.

Reply:
Or you may just `s.NoError(err)`

Reply:
I thought we should use `require.NoError` but `s.Noerror` is a shortcut as `assert.NoError`.